api/v1beta1/cryostat_types.go

```diff
@@ -517,7 +517,7 @@ type SecurityOptions struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	StorageSecurityContext *corev1.SecurityContext `json:"storageSecurityContext,omitempty"`
-	// Security Context to apply to the storage container.
+	// Security Context to apply to the database container.
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	DatabaseSecurityContext *corev1.SecurityContext `json:"databaseSecurityContext,omitempty"`
```

Comment:
Just noticed these new specs should only be available on `v1beta2`? The `v1beta1` API should be the one from `2.x` releases?